Modules/Devices/src/Devices.Infrastructure.Database.SqlServer/Migrations/20230413132800_IdentityBasicTier.cs

```diff
@@ -29,7 +36,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 table: "Identities",
                 column: "TierId",
                 principalTable: "Tier",
-                principalColumn: "Id");
+                principalColumn: "Id",
+                onDelete: ReferentialAction.Cascade);
```

Comment:
This is evil. This means that if we delete a Tier, all Identities assigned to the Tier will be deleted.

Reply:
Indeed! Should we set those as Null, or restrict the delete? We do not have a default value.

Reply:
Since we don't want to have Identities without a Tier, we should go with Restrict.